### PR TITLE
Iconography editorial for pull #62

### PR DESCRIPTION
**Pull request by designakt**
editorial for https://github.com/bwinton/DesignSystem/pull/62

### CODE REVIEW

**Comment by designakt** [comment]
The text contains many instructions on how to use software (Sketch and Adobe Illustrator). This text seams to be not directly contributing to the design system, but is rather educational for users. @brassy- Can you please distinguish those parts from the design system content. Maybe `<article>` or `<blockquote>` could be of use.

File: contents/iconography.html [hunk 1]
```diff
@@ -6,22 +6,22 @@
...
+    A good Firefox icon is simple but still representing the right metaphor. Use universal metaphors that everyone around the world can quickly identify. Aim for a design most people can interpret correctly and won’t find offensive. Do not focus on a secondary or obscure aspect of an element. Remember that too many details can make an icon appear sloppy or unreadable.
   </p>
   <h3>Grid</h3>
   <p>
-    Firefox icons live on a 16x16 pixel grid. Keep your shape weighted towards the middle area of the grid reserving the outer 1px margin for visual weighting and maintaining relative aspect ratios. Designing with the pixel grid on help the icons not appear fuzzy in production.
+    Firefox icons live on a 16&times;16 pixel grid. Keep your shape weighted towards the middle area of the grid reserving the outer 1px margin for visual weighting and maintaining relative aspect ratios. Designing on awith the pixel grid on helps ensure that the icons will not appear fuzzy in production.
```
Comment (aminalhazwani) [inline contents/iconography.html]: Not sure what do you mean with "on awith" 🤔

File: contents/iconography.html [hunk 2]
```diff
@@ -6,22 +6,22 @@
...
   </p>
   <p>
-    In Illustrator can turn on the pixel grid from the Transform Panel: in the options menu just tick on <code>Align new objects</code> to pixel grid feature.
+    In Sketch, select the shape and then select <em>Layer &#8594; Round to Nearest Pixel Edge</em>. Sketch will automatically align the edges with the pixel grid. It works for both the Position and Size values.
   </p>
   <p>
-    In Sketch, select the shape and then select <code>Layer &#8594; Round to Nearest Pixel Edge</code>. Sketch will automatically align the edges with the pixel grid. It works for both the Position and Size values. 
+    In Adobe Illustrator, use <em>View &#8594; Pixel Preview</em> to see you design on a pixel grid. Select a shape and use <em>Object &#8594; Make Pixel Perfect</em> to set position and size to the nearest full pixel. Or click the <em>Snap to Pixel</em> icon in the Control panel to turn on snapping permanently.
```
Comment (aminalhazwani) [inline contents/iconography.html]: [...] to see youR design... I believe

**Comment by aminalhazwani** [comment]
Merged `iconography-editorial` into `icons-production`. This pull request will automatically close once we merge `icons-production` into master 😄

**Comment by aminalhazwani** [comment]
Merged `icons-production` into `production`, see [ebdb46](https://github.com/bwinton/DesignSystem/commit/ebdb460f118ca3c98d801d4d0a1fed5cc28942c7)

**Pull request closed without merging**